src/jetstream/mod.rs

```diff
@@ -693,6 +696,89 @@ impl JetStream {
         self.do_push_subscribe(subject, None, None)
     }
 
+    /// Creates a `PullSubscription`.
+    pub fn pull_subscribe(&self, subject: &str, consumer: &str) -> io::Result<PullSubscription> {
```

Comment:
@derekcollison  After NATS server 2.7.0 `consumer` should become optional, as we now have ephemeral pull  consumers, right?

Reply:
We need to figure out how to present that option at the client layer but yes.